data_model/1.4/clusters/DoorLock.xml

```diff
@@ -125,10 +124,10 @@ Davis, CA 95616, USA
     <feature bit="12" code="UBOLT" name="Unbolting" summary="Lock supports unbolting">
       <optionalConform/>
     </feature>
-    <feature bit="13" code="ALIRO" name="AliroProvisioning" summary="Lock supports Aliro credential provisioning as defined in ref_Aliro">
+    <feature bit="13" code="ALIRO" name="AliroProvisioning" summary="Lock supports Aliro credential provisioning as defined in [Aliro&amp;#93;">
```

Comment:
This is surely better than the "ref_" but it still looks a bit weird?

Reply:
A bit. I'm still good with it since I don't use the summaries for anything currently.